pom.xml

```diff
@@ -34,7 +34,7 @@
   </licenses>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <selenium.version>2.53.0</selenium.version> <!-- note: selenium does not follow semantic versioning -->
+    <selenium.version>3.5.1</selenium.version> <!-- note: selenium does not follow semantic versioning -->
```

Comment:
Is this supposed to be 3.7.1?

Reply:
@scrubious, since this version of {{selenium}} {{MutableCapabilities}} are available.

Reply:
Please, let me provide more details on this. We tried to use JBrowserDriver with Selenium 3.7.1. But it was discovered that they are not compatible, it was needed to introduce some fixes (see actual changes). At the same time the upgrade of Selenium version was required, but also it would be great to keep some compatibility with old versions of Selenium. So Selenium 3.5.1 is the lowest version which is needed to introduce changes and to run successfully on Selenium 3.7.1.